features/atcryptoauth/sample_ssl_client.cpp

```diff
@@ -0,0 +1,380 @@
+#else
+#include <stdio.h>
+#include <stdlib.h>
+#define mbedtls_time       time 
+#define mbedtls_time_t     time_t
+#define mbedtls_fprintf    fprintf
+#define mbedtls_printf     printf
+#endif /* MBEDTLS_PLATFORM_C */
+
+#ifdef SSL_CLIENT_APP_ACTIVATE_DEBUG
+#include "mbedtls/debug.h"
+#endif
+#include "mbedtls/ssl.h"
+#include "mbedtls/entropy.h"
+#include "mbedtls/ctr_drbg.h"
+#include "mbedtls/error.h"
+#include "mbedtls/certs.h"
+#include "mbedtls/pk_info.h"
+#include "mbedtls/net_sockets.h"
+
+/* Mbed OS includes */
+#include "mbed.h"
+#include "EthernetInterface.h"
+#include "mbedtls_atca_engine.h"
+
+#define SERVER_PORT 4433
+#define SERVER_NAME "E112111.emea.arm.com"
+
+#define GET_REQUEST "GET / HTTP/1.0 \r\n\r\n"
+
+#ifdef SSL_CLIENT_APP_ACTIVATE_DEBUG
+#define DEBUG_LEVEL 255
+#endif /* SSL_CLIENT_APP_ACTIVATE_DEBUG */
+
+
+static const char client_cert_pem[] = 
+"-----BEGIN CERTIFICATE-----\r\n"
+"MIIBfTCCASGgAwIBAgIBATAMBggqhkjOPQQDAgUAMC8xDTALBgNVBAMTBENlcnQx\r\n"
+"ETAPBgNVBAoTCG1iZWQgVExTMQswCQYDVQQGEwJVSzAeFw0wMTAxMDEwMDAwMDBa\r\n"
+"Fw0zMDEyMzEyMzU5NTlaMC8xDTALBgNVBAMTBENlcnQxETAPBgNVBAoTCG1iZWQg\r\n"
+"VExTMQswCQYDVQQGEwJVSzBZMBMGByqGSM49AgEGCCqGSM49AwEHA0IABOC03rQc\r\n"
+"nePEsBd8IVKaBStz2LfRNAXnxbyR7vbdP2qvF72KHfcwj3/EieTz7o0v399J9xBe\r\n"
+"Gjviq0DPMLRYpq6jLDAqMAkGA1UdEwQCMAAwHQYDVR0OBBYEFG973jHQ2GP8JXd2\r\n"
+"9HUvNT2GWZV2MAwGCCqGSM49BAMCBQADSAAwRQIgP9Dv55CBaj2ZE+XVxkUizE0A\r\n"
+"GsdZvOiFnlOICCOyB4gCIQCFqRLUkctB3Anxfqz6E3hjlGk68i5PNnXDvcRkVzQE\r\n"
+"Wg==\r\n"
+"-----END CERTIFICATE-----\r\n";
+
+static const char ca_cert_pem[] =
+"-----BEGIN CERTIFICATE-----\r\n"
+"MIICfDCCAiGgAwIBAgIBADAKBggqhkjOPQQDAjCBnTELMAkGA1UEBhMCR0IxFzAV\r\n"
+"BgNVBAgMDkNhbWJyaWRnZXNoaXJlMRIwEAYDVQQHDAlDYW1icmlkZ2UxDjAMBgNV\r\n"
+"BAoMBU15IENvMRAwDgYDVQQLDAdNeSBEZXB0MR0wGwYDVQQDDBRtYXppbWtoYW4t\r\n"
+"VmlydHVhbEJveDEgMB4GCSqGSIb3DQEJARYRQXppbS5LaGFuQGFybS5jb20wHhcN\r\n"
+"MTgwMjAzMDEzMzA3WhcNMjgwMjAxMDEzMzA3WjCBnTELMAkGA1UEBhMCR0IxFzAV\r\n"
+"BgNVBAgMDkNhbWJyaWRnZXNoaXJlMRIwEAYDVQQHDAlDYW1icmlkZ2UxDjAMBgNV\r\n"
+"BAoMBU15IENvMRAwDgYDVQQLDAdNeSBEZXB0MR0wGwYDVQQDDBRtYXppbWtoYW4t\r\n"
+"VmlydHVhbEJveDEgMB4GCSqGSIb3DQEJARYRQXppbS5LaGFuQGFybS5jb20wWTAT\r\n"
+"BgcqhkjOPQIBBggqhkjOPQMBBwNCAAS+ImO4q+2w6l8fN3y9dRFgmb4dmaCYIUK6\r\n"
+"ZG0hGT/AqLC8ds0al7Mzw8+MUsYNpsxUToFU4EvV3iCxhd2vZFhho1AwTjAdBgNV\r\n"
+"HQ4EFgQUBN3Eifs+3C9ZwHMAF3GBPNKDH08wHwYDVR0jBBgwFoAUBN3Eifs+3C9Z\r\n"
+"wHMAF3GBPNKDH08wDAYDVR0TBAUwAwEB/zAKBggqhkjOPQQDAgNJADBGAiEAhYwb\r\n"
+"wAQiGkOTt/YbdPY1ZuEjCBY6dyqCPI9blT0EVvQCIQCJCSLSQ0IW37SI4RHXuBDZ\r\n"
+"fCh27v3WWb8djrwpjSyg+g==\r\n"
+"-----END CERTIFICATE-----\r\n";
+
+static TCPSocket * _tcpsocket = NULL;
+
+int mbedtls_net_init()
+{
+    EthernetInterface eth_iface;
+
+    mbedtls_printf( "\r\n************************* GET IP ADDRESS *************************\r\n" );
+    eth_iface.connect();
+    const char *ip_addr = eth_iface.get_ip_address();
+    if (ip_addr)
+        mbedtls_printf("Client IP Address is %s\r\n", ip_addr);
+    else
```

Comment:
I assume this is mbed OS file, thus should follow the style . Same of the rest of files in features/atcryptoauth